app/vmselect/prometheus/prometheus.go

```diff
@@ -61,9 +61,15 @@ const defaultStep = 5 * 60 * 1000
 // ExpandWithExprs handles the request to /expand-with-exprs
 func ExpandWithExprs(w http.ResponseWriter, r *http.Request) {
 	query := r.FormValue("query")
+	json := r.FormValue("json")
```

Comment:
As I understand, the json field is set by VMUI to get response in a specific format. 
Why not use `Content-Type` header in client's call? I thought it is a common practice to signalize in what format you expect to get the response.

Reply:
Good point. I think we can fix this part

Reply:
I disagree - it is easier to debug queries by passing `format=json` query arg in url instead of specifying `Content-Type` request header. Let's just rename the query arg to `format`, since `json` isn't an obvious name for the query arg.

Reply:
@valyala your comment was fixed, please take a look